### DISCUSSION TITLE
Why is the number of x-coordinates inconsistent with the number of y-coordinates in the NASA_R67.cfg file

### DISCUSSION THREAD

**Discussion by wuyueteng** [NAnand-TUD/parablade#22] [General]
Hello, I am delighted to see your excellent software, which has been very helpful for my work. However, I have a confusion while using NASA_R67. Why is the number of x-coordinates inconsistent with the number of y-coordinates in the .cfg file? Could you please help me with this question? I am looking forward to your reply.If possible, could you write a software user manual? This would greatly help with the promotion of the software.
![NASA_R67](https://github.com/NAnand-TUD/parablade/assets/93500954/6c66ae20-aaa4-49e0-b522-a9a2677017de)

**Comment by xiapxiaoyu1981** [comment]
Hi Wu, are you still using ParaBlade? I've been studying this code recently and would love to exchange contact information with you.
